OxfordBodleian/Juel-Jensen/BDLaethd19.xml

```diff
@@ -184,45 +183,102 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                                        <dim type="right">35</dim>
-                                        <dim type="left">15</dim>
-                                    </dimensions>
-                                </layout>
 
-                                <layout columns="2" writtenLines="24">
-                                    <locus from="148ra" to="165vb"/>
-                                    <note corresp="#textarea2 #margin2">Data on text area and margin dimensions taken from <locus target="#151r"/>.</note>
-                                    <dimensions unit="mm" xml:id="textarea2">
-                                        <height>207</height>
-                                        <width>155</width>
-                                    </dimensions>
-                                    <dimensions type="margin" unit="mm" xml:id="margin2">
-                                        <dim type="top">30</dim>
-                                        <dim type="bottom">50</dim>
-                                        <dim type="left">10</dim>
-                                        <dim type="right">40</dim>
-                                        <dim type="intercolumn">13</dim>
-                                    </dimensions>
-                                </layout>
-                            </layoutDesc>
                         </objectDesc>
 
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-
+                                <locus from="1r" to="165ra"/>
+                                <seg type="ink">Black, red.</seg>
+                                <date notBefore="1400" notAfter="1499"/> ("middle of the fifteenth century", 
+                                <bibl><ptr target="bm:JuelJensen1994Bindings"/><citedRange unit="page">186</citedRange></bibl>)
+                                <desc>
+                                    Fine and trained hand. Characters are regularly spaced.
+                                    The script displays some relatively archaic features: the numeral <foreign xml:lang="gez">፮</foreign> has 
+                                    a compressed shape and the open loop;
+                                    the letter <foreign xml:lang="gez">መ</foreign> has the loops not completely separated; the 
+                                        letter <foreign xml:lang="gez">ዐ</foreign> and 
+                                    numeral <foreign xml:lang="gez">፬</foreign> show "cone-forms" (for the letter 
+                                    <foreign xml:lang="gez">ፀ</foreign>, this is much less 
+                                    salient); the letter <foreign xml:lang="gez">ኍ</foreign> has the vowel marker on the top of the letter body; 
+                                    however, the word <foreign xml:lang="gez">እግዚአ፡ ብሔር፡</foreign> is written as one word. Although the 
```

Comment:
If እግዚአ፡ ብሔር፡ is written as one word, maybe it would be better to use the same spelling here